libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -729,12 +736,43 @@ void AP_MotorsUGV::output_skid_steering(bool armed, float steering, float thrott
     float steering_scaled = steering / 4500.0f; // steering scaled -1 to +1
     float throttle_scaled = throttle * 0.01f;  // throttle scaled -1 to +1
 
+    // sanitize values for asymmetry of thrust, mixer assumes forward thrust is always larger than reverse
+    const float thrust_asymmetry = MAX(_thrust_asymmetry, 1.0);
+    const float min_throttle = -1.0 / thrust_asymmetry;
```

Comment:
We have a very similarly named parameter _throttle_min so I was thrown off for a moment by "min_throttle".  Not a blocker but if we can think of another name that might be nice.

Reply:
Changed to `lower_throttle_limit`. Its different but still not great, I'm open to suggestions.